src/main/scala/jtag/JtagStateMachine.scala

```diff
@@ -66,7 +66,7 @@ object JtagState {
   * Usage notes:
   * - 6.1.1.1b state transitions occur on TCK rising edge
   * - 6.1.1.1c actions can occur on the following TCK falling or rising edge
-  *
+  * Implicit reset is AsyncReset
```

Comment:
there is now a `RequireAsyncReset` mixin we could use to enforce this below (though we would need to get up to date on chisel):

`class JtagStateMachine ... extends Module with RequireAsyncReset`

Reply:
Will add when available.